libusb/include/usbdriver.h

```diff
@@ -146,13 +167,20 @@ typedef struct {
 } usb_modeswitch_t;
 
 
+typedef struct {
+	bool deviceCreated;
```

Comment:
Could you clarify the the purpose of this field?

Reply:
It is set to true by the device driver if it creates a device in devfs for the usb device it has inserted. Then, the dev and devPath fields point to this device's oid and path in the fs

Reply:
Added a comment